docs/modules/rsem.md

```diff
@@ -0,0 +1,27 @@
+---
+Name: rsem
+URL: https://deweylab.github.io/RSEM/
+Description: >
+    RSEM (RNA-Seq by Expectation-Maximization) is a software package for 
+    estimating gene and isoform expression levels from RNA-Seq data. 
+---
+
+The rsem module parses results generated by
+[RSEM](https://deweylab.github.io/RSEM/),
+a software package for estimating gene and isoform expression levels 
+from RNA-Seq data
+
+Supported scripts:
+
+* `rsem-calculate-expression`
+
+You can choose to hide sections of RSEL output and customise their order.
+To do this, add and customise the following to your MultiQC config file:
+
+```yaml
+sp:
```

Comment:
This example changes the search that MultiQC uses to find rsem files, rather than hiding sections or customising their order..?